pkg/deploy/generator/resources_rp.go

```diff
@@ -1518,36 +1518,13 @@ func (g *generator) rpACRRBAC() []*arm.Resource {
 
 func (g *generator) rpVersionStorageAccount() []*arm.Resource {
 	return []*arm.Resource{
-		g.storageAccount("[parameters('rpVersionStorageAccountName')]", &mgmtstorage.AccountProperties{
-			AllowBlobPublicAccess: to.BoolPtr(true),
-		}, map[string]*string{
-			tagKeyExemptPublicBlob: to.StringPtr(tagValueExemptPublicBlob),
-		}),
-		{
-			Resource: &mgmtstorage.BlobContainer{
-				Name: to.StringPtr("[concat(parameters('rpVersionStorageAccountName'), '/default/rpversion')]"),
-				Type: to.StringPtr("Microsoft.Storage/storageAccounts/blobServices/containers"),
-				ContainerProperties: &mgmtstorage.ContainerProperties{
-					PublicAccess: mgmtstorage.PublicAccessContainer,
-				},
-			},
-			APIVersion: azureclient.APIVersion("Microsoft.Storage"),
-			DependsOn: []string{
-				"[resourceId('Microsoft.Storage/storageAccounts', parameters('rpVersionStorageAccountName'))]",
-			},
-		},
-		{
-			Resource: &mgmtstorage.BlobContainer{
-				Name: to.StringPtr("[concat(parameters('rpVersionStorageAccountName'), '/default/ocpversions')]"),
```

Comment:
I'm fairly confident we're not using OCPVersions at all, but can you confirm?  I'm pretty certain they're stale.

Reply:
For context/history (if i'm remembering correctly), we used this back when we had a single version install target and didn't support multiple versions.

Reply:
I checked our prod container and the newest OCP version it had was a 4.11.z version, so I hope we still aren't using it for anything

Reply:
It's also worth noting that the absence of these blobcontainer resources doesn't result in existing containers being deleted, they're just left alone (verified in int). If we do want to delete these resources we'll likely need to add an explicit fixup step during deployment.